dashboard/app/controllers/foorm/library_questions_controller.rb

```diff
@@ -5,20 +5,85 @@ class LibraryQuestionsController < ApplicationController
     before_action :authenticate_user!
     load_and_authorize_resource
 
+    # POST /foorm/library_questions
+    def create
+      library = Foorm::Library.find_by(id: params[:library_id])
```

Comment:
I think you can just use `find(params[:library_id])` instead of `find_by(id: params[:library_id])`